rest/data/project_settings_test.go

```diff
@@ -561,6 +561,206 @@ func TestSaveProjectSettingsForSection(t *testing.T) {
 	}
 }
 
+func TestPromoteVarsToRepo(t *testing.T) {
+	for name, test := range map[string]func(t *testing.T, ref model.ProjectRef){
+		"SuccessfullyPromotesAllVariables": func(t *testing.T, ref model.ProjectRef) {
+			projectId := "pId"
+			repoId := "rId"
+
+			varsToPromote := []string{"a", "b", "c"}
+			err := PromoteVarsToRepo(projectId, varsToPromote, "u")
+			assert.NoError(t, err)
+
+			projectVarsFromDb, err := model.FindOneProjectVars(projectId)
```

Comment:
nit: from [the style guide](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.3rcuk9z7pv53), prefer to case this variable as `DB` instead of `Db` to follow acronym capitalization.